src/api.ts

```diff
@@ -0,0 +1,16 @@
+const API_URL = 'https://jsonplaceholder.typicode.com';
+
+const getItems = async<T>(url: string): Promise<T[]> => {
+  const response = await fetch(API_URL + url);
+
+  if (!response.ok) {
+    return [];
+  }
+
+  return response.json()
+    .catch(() => []);
```

Comment:
it is better to create a separate variable and write the result of the query to it
let items = await response.json()
return items